testnet/launcher/l1grantsequencers/docker.go

```diff
@@ -18,14 +27,25 @@ func NewGrantSequencers(cfg *Config) (*GrantSequencers, error) {
 }
 
 func (s *GrantSequencers) Start() error {
+	var enclaveIDs string
+	var err error
+	if s.cfg.enclaveIDs != "" {
+		enclaveIDs = s.cfg.enclaveIDs
+	} else if s.cfg.sequencerURL != "" {
+		enclaveIDs, err = fetchEnclaveIDs(s.cfg.sequencerURL)
+		if err != nil {
+			return err
+		}
+	} else {
+		return fmt.Errorf("enclaveIDs or sequencerURL must be provided")
+	}
 	cmds := []string{
 		"npx",
+		"hardhat",
```

Comment:
This script didn't seem to be running properly, it needed the hardhat command to use the --network flag and I couldn't get it to work with positional args so changed them to env vars.